### ISSUE TITLE
[0.1.0] Unmet peer dependency error where none is missing

### ISSUE THREAD

**Issue by nikwen** [FortAwesome/vue-fontawesome#89]
I ran:

```
npm i @fortawesome/fontawesome-svg-core@prerelease \
      @fortawesome/free-solid-svg-icons@prerelease \
      @fortawesome/vue-fontawesome@prerelease
```

npm yields:

`npm WARN @fortawesome/vue-fontawesome@0.1.0-4 requires a peer of @fortawesome/fontawesome-svg-core@>= 1.2.0 < 1.3 but none is installed. You must install peer dependencies yourself.`

Excerpt from package.json:

```
"dependencies": {
    "@fortawesome/fontawesome-svg-core": "1.2.0-14",
    "@fortawesome/free-solid-svg-icons": "5.1.0-11",
    "@fortawesome/vue-fontawesome": "0.1.0-4",
    ...
}
```

Might be related to #60.

**Comment by robmadole** [comment]
Yes, this is an annoying thing with pre-releases. The range cannot be specified correctly. Doing `>=1.2.0-0 <1.3` doesn't catch them all. (Used https://semver.npmjs.com to test this).

I think we'll live with this one until we release the final version.

**Comment by nikwen** [comment]
Thanks for the quick reply!

I had a look into it and it seems as if the prerelease naming scheme used by fontawesome is incorrect according to the npm semver documentation: https://docs.npmjs.com/misc/semver#prerelease-tags

Prerelease tags need to contain a prefix before the prerelease number, e.g. `1.2.0-alpha.7`. Then a range such as `>=1.2.0-alpha.0 <1.3` should work. (You can test it on the `lodash` package with a range such as `>= 1.0.0-rc.2 <1.1`.)

So the proper fix would be for fontawesome to fix their version numbers and align them with the npm semver docs.

**Comment by robmadole** [comment]
Yeah we probably need to change this. We chose the current scheme based on https://semver.org/#spec-item-9. I wonder if our issues could be considered a bug in the `semver` package.

**Comment by nikwen** [comment]
Please change it. It would make using prereleases a lot easier and you are always encouraging people to do so. :+1:

**Comment by robmadole** [comment]
We aren't going to change it for this round of pre-releases. We'll be releasing the full versions soon.

**Comment by eliteproxy7** [comment]
has this been corrected yet?

**Comment by nikwen** [comment]
@eliteproxy7 It's been "corrected" in the 0.1.0 stable release in the sense that this was an issue with prerelease versioning. I don't know if it has been fixed for future prereleases already.

@robmadole Can you comment on that?

**Comment by robmadole** [comment]
We don't have a future pre-release yet. However, I don't think we're going to use ranges with prerelease versions any longer (they just are not predictable between tools). So the underlying issue for this won't be an issue in the future.
